src/common_tasks/transform_data.py

```diff
@@ -24,41 +24,33 @@ def df_field_strip(dataframe: pd.DataFrame, field_name: str, chars_to_remove: st
         dataframe[field_name] = dataframe[field_name].apply(lambda x: x.strip(chars_to_remove) if x else x)
     return dataframe
 
-class HtmlToText():
-    def __init__(self):
-        h = html2text.HTML2Text()
-        h.ignore_links = True
-        h.ignore_images = True
-        h.ignore_emphasis = True
-        h.body_width = 1024
-        h.strong_mark = ''
-        h.emphasis_mark = ''
-        h.ul_item_mark = ''
 
-        re_dummy = re.compile('(.^)(.^)')
-        html2text.config.RE_MD_BACKSLASH_MATCHER = re_dummy
-        html2text.config.RE_MD_DOT_MATCHER = re_dummy
-        html2text.config.RE_MD_PLUS_MATCHER = re_dummy
-        html2text.config.RE_MD_DASH_MATCHER = re_dummy
+class MLRemover(HTMLParser):
```

Comment:
didn't get what ML means here at first. Maybe rename it to something like MarkupRemover?

Reply:
Suggestion applied.